cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -276,10 +251,12 @@ public void start(ProgramId programId, Map<String, String> overrides, boolean de
    * @return {@link ProgramRuntimeService.RuntimeInfo}
    * @throws IOException if there is an error starting the program
    * @throws ProgramNotFoundException if program is not found
-   * @throws UnauthorizedException if the logged in user is not authorized to start the program
+   * @throws UnauthorizedException if the logged in user is not authorized to start the program. To stop a program,
```

Comment:
Should the comment say `if the logged in user is not authorized to start the program. To start a program`?